source/gear/feverbreak-surpassing.js

```diff
@@ -0,0 +1,40 @@
+const { GearTemplate } = require('../classes');
+const { dealDamage, removeModifier } = require('../util/combatantUtil');
+const { listifyEN } = require('../util/textUtil');
+const { SURPASSING_VALUE } = require('../constants');
+
+module.exports = new GearTemplate("Surpassing Fever Break",
+    "Deals to a foe, the damage that is pending from any Poison and Frail on them, and then removes those debuffs",
```

Comment:
Don't forget to update the description to state what the Surpassing variant does. You can use `@{bonus}` to interpolate the value you provided with `.setBonus()`.